clojure/README.md

```diff
@@ -0,0 +1,43 @@
+# clojure
+
+Clojure is a robust, practical, and fast programming language with a set of useful features that together form a simple, coherent, and powerful tool.
+
+## Maintainers
+
+* The Habitat Maintainers: <humans@habitat.sh>
+
+## Type of Package
+
+Binary package
+
+## Usage
+
+Install:
+```
+hab pkg install core/clojure
+```
+
+Execute:
+```
+hab pkg exec core/clojure clj -h
+```
+
+Add to a plan:
+```
+pkg_deps=(
+  core/clojure
+)
+```
+
+Start an nrepl:
+```
+hab svc load core/clojure
+hab svc start core/clojure
+```
+
```

Comment:
To validate the running nrepl service is running, it might be nice to give the command to connect to the nrepl at this point e.g.

```
hab pkg exec core/clojure clj -Sdeps '{:deps {nrepl {:mvn/version "0.6.0"}}}' -m nrepl.cmdline --connect --host localhost --port 31337
```

Reply:
@james-stocks: added this command to README